src/cloudwatch/modules/client/ec2getclient.py

```diff
@@ -0,0 +1,82 @@
+import re
+
+from ..plugininfo import PLUGIN_NAME, PLUGIN_VERSION
+from ec2requestbuilder import EC2RequestBuilder
+from ..logger.logger import get_logger
+from requests.adapters import HTTPAdapter
+from requests.sessions import Session
+import xml.etree.ElementTree as ET
+from querystringbuilder import QuerystringBuilder
+
+
+
+class EC2GetClient(object):
```

Comment:
Is there a reason we're not just using the AWS SDK to get this information? Am I missing something really obvious about why this isn't done in collectd plugins?